spec/unit/files/servicenow_spec.rb

```diff
@@ -6,11 +8,12 @@
 
 describe 'servicenow' do
   let(:api_response) { File.read('./spec/support/files/valid_api_response.json') }
-  let(:config) { JSON.parse(File.read('./spec/support/files/default_config.json')) }
+  let(:config) { YAML.load_file('./spec/support/files/default_config.yaml') }
```

Comment:
Had to change this so `YAML.load` wouldn't file. Also good to be consistent -- code reads a YAML file so the tests should also read a YAML file.

Reply:
Ended up updating the implementation so am still using `YAML.load_file`. However I do like being consistent in tests (code reads from YAML file so test reads from YAML file), so I'd like to keep this change.